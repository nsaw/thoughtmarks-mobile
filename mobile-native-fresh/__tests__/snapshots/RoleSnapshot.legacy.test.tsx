import React from 'react';
import renderer from 'react-test-renderer';

// Simple test component
const TestCard = ({ children }: { children: React.ReactNode }) => (
  <div data-testid="card">{children}</div>
);

const TestTagChip = ({ label }: { label: string }) => (
  <span data-testid="tag-chip">{label}</span>
);

describe('RoleSnapshot JSX Structure', () => {
  it('renders <TestCard /> correctly', () => {
    const tree = renderer.create(<TestCard>Snapshot JSX Test</TestCard>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders <TestTagChip /> correctly', () => {
    const tree = renderer.create(<TestTagChip label="RoleTag" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
}); 