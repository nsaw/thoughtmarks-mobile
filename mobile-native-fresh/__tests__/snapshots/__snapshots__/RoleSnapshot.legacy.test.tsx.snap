// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoleSnapshot JSX Structure renders <TestCard /> correctly 1`] = `null`;

exports[`RoleSnapshot JSX Structure renders <TestTagChip /> correctly 1`] = `null`;
